rust/worker/src/execution/operators/metadata_filtering.rs

```diff
@@ -0,0 +1,1348 @@
+use core::panic;
+use std::{
+    collections::{HashMap, HashSet},
+    sync::{
+        atomic::{AtomicU16, AtomicU32},
+        Arc,
+    },
+};
+
+use futures::stream::Count;
+use roaring::RoaringBitmap;
+use thiserror::Error;
+use tonic::async_trait;
+
+use crate::{
```

Comment:
nit: please make use statements have no whitespace between them, i can see if there is a linter rule we can use here.

Reply:
Yeah I have never added a use statement by myself ever. Rust analyzer seems to add these new lines